dataland-frontend/src/components/forms/parts/elements/derived/ProcurementCategoryFormElement.vue

```diff
@@ -0,0 +1,177 @@
+<template>
+  <div class="form-field">
+    <div data-test="dataPointToggle" class="form-field border-none vertical-middle">
+      <InputSwitch
+        data-test="dataPointToggleButton"
+        inputId="dataPointIsAvailableSwitch"
+        @click="this.isItActive = false"
+        v-model="isItActive"
+      />
+      <h5 data-test="dataPointToggleTitle" class="m-2">
+        {{ label }}
+      </h5>
+    </div>
+
+    <FormKit type="group" :name="name" v-if="isItActive">
+      <div>
+        <div class="form-field border-none">
+          <NaceCodeFormFieldBindData
```

Comment:
I don't understand why it is not possible to use the `NaceCodeFormField.vue` component that we already have.
Could you explain it pls?

Reply:
Ok thanks for the explanation. Now I understand it:
The parent component (ProcurementCategoryFormElement.vue) is something like a "storage" for the value of `selectedNaceCodes`. So if you untoggle/deactivate a procurement category and toggle/reactivate it again, the nace codes are still there, because they were kept in the data variable `procuredProductTypesAndServicesNaceCodesValue`.
Is this correct?

So if you used our existing `NaceCodeFormField.vue` here, the only difference would be, that the selected nace codes are gone if you untoggle a procurement category.
Besides that, it would work.
Correct?

Reply:
Yes, exactly

Reply:
Ok then it's fine for me and a cool solution. Thx again for the explanation 👍